### PR TITLE
fixed README for VMSS templates

### PR DESCRIPTION
**Pull request by gatneil**
fixed a couple errors in READMEs for VMSS templates

### CODE REVIEW

File: 301-multi-vmss-linux/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                2,
+                3,
+                4,
+                5
+            ]
+        },
+        "instanceCountPerVMSS": {
+            "type": "string",
+            "metadata": {
+                "description": "Number of VM instances per scale set"
```
Comment (singhkay) [inline 301-multi-vmss-linux/azuredeploy.json]: Probably needs a max allowed value for this

File: 301-multi-vmss-linux/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            }
+        },
+        "newVNETName": {
+            "type": "string"
+        },
+        "newStorageAccountPrefix": {
+            "type": "string"
```
Comment (singhkay) [inline 301-multi-vmss-linux/azuredeploy.json]: thought we were auto-gen'd the storage names?

File: 301-multi-vmss-linux/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        "addressPrefix": "10.0.0.0/16",
+        "subnetName": "subnet",
+        "subnetPrefix": "10.0.0.0/16",
+        "storageAccountType": "Standard_LRS",
+        "baseTemplateUri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-multi-vmss-linux/base.json",
+        "location": "[resourceGroup().location]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-05-01-preview",
```
Comment (singhkay) [inline 301-multi-vmss-linux/azuredeploy.json]: Should be a variable and used in other templates as well

File: 301-multi-vmss-linux/azuredeploy.json (hunk 1 above)
Comment (gatneil) [inline 301-multi-vmss-linux/azuredeploy.json]: Instead of allowedValues, or in addition to?

Comment (singhkay) [inline 301-multi-vmss-linux/azuredeploy.json]: Meant maxValue like in the other template. See this https://github.com/Azure/azure-quickstart-templates/blob/master/201-vmss-linux-jumpbox/azuredeploy.json#L59

**Pull request merged**